.github/workflows/build_and_test.yml

```
@@ -242,6 +256,13 @@ jobs:
           source ${{ env.BUILD_ROOT }}/kong-dev-venv.sh
           .ci/run_tests.sh
 
+    - name: Upload test rerun information
+      if: always()
+      uses: actions/upload-artifact@v3
+      with:
+        name: ${{ env.FAILED_TEST_FILES_FILE }}
+        path: ${{ env.FAILED_TEST_FILES_FILE }}
```

```suggestion
        path: ${{ env.FAILED_TEST_FILES_FILE }}
        retention-days: 2
```

We don't need to keep this artifact for too long, especially since it's not free for private repo.

---

@ADD-SP I found we set 3 retention days for most actions. Any reasons we want it to be 2 here?